README.md

```diff
@@ -451,17 +452,19 @@ In general, startups rely on investors to help fund rapid growth.
 🄳 In order to finance building or scaling their business, startups **fundraise** by selling
 shares in their business to [investors](http://www.investopedia.com/terms/i/investor.asp) in
 exchange for capital.
-Startups that aspire to grow rapidly are likely to fundraise from
-[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital) investors, who are
-individuals or firms that specialize in such investments.
-After a company IPOs and is public, it can seek investment in public markets.
+Startups that aspire to grow rapidly are likely to fundraise from individuals or firms
+specializing in startup investment, called
+[**venture capital**](https://en.wikipedia.org/wiki/Venture_capital). After a company goes public
+in what is known as an [initial public offering (IPO)](#ipos), it can seek investment in
+public markets.
 
 A startup goes through [several stages of growth](#stages-of-a-startup) as it fundraises
 based on evidence that it will make money in the future.
 
-- 🄳 As these companies add (or “issue”) stock, the number of outstanding shares goes up,
-  and the percentage ownership of each shareholder goes down.
-  This is called [**dilution**](http://www.investopedia.com/terms/d/dilution.asp).
+- 🄳 Companies add (or “issue”) shares during fundraising, creating more pieces of the
```

Comment:
Companies add (or “issue”) shares during fundraising, which can then be awarded to investors in exchange for their investment.

Reply:
Fixed. (I do find this language slightly misleading. You can't technically be "awarded" something that you've purchased, as awards are by definition not part of an exchange. The "award" part sounds like an extra gift only some investors receive. Don't only employees receive stock awards?)